packages/pluggableWidgets/app-events-native/src/AppEvents.xml

```diff
@@ -10,6 +10,10 @@
                     <caption>On load</caption>
                     <description>This action is immediately triggered when this widget is created.</description>
                 </property>
+<property key="onUnloadAction" type="action" required="false">
```

Comment:
Indentation is wrong

Reply:
Please fix this to make us able to merege

Reply:
fixed in 6511e7c